### DISCUSSION TITLE
How to update Snapshot for specific test instead of all tests

### DISCUSSION THREAD

**Discussion by 787681277** [cypress-io/cypress#23795] [Questions and Help]
In the "cypress-image-snapshot" doc,
--env updateSnapshots=true in order to update the base image files for all of your tests.

But How to update Snapshot for specific test instead of all test?

**Comment by emilyrohrbough** [comment]
The `cypress-image-snapshot` plugin is not maintained by Cypress. Please ask on their repo: https://www.npmjs.com/package/cypress-image-snapshot or each out to the community in discord: https://on.cypress.io/chat.
